api/gateway/middleware.go

```diff
@@ -41,7 +41,7 @@ func checkPostDisabled(state state.Module) mux.MiddlewareFunc {
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			// check if state service was halted and deny the transaction
-			if r.Method == http.MethodPost && state.IsStopped(r.Context()) {
+			if r.Method == http.MethodPost {
```

Comment:
in this case, POST requests will be disabled.

Reply:
Yeah we only have one `POST` endpoint left and it's `SubmitTx` which is safe to leave. 

let's just remove this check altogether as it's no longer necessary - if there is no state service running, it will error out regardless (it's just that the error might not be so nice for now).